Docs/source/usage/parameters.rst

```diff
@@ -1265,6 +1265,11 @@ Laser initialization
     per angular mode. The laser particles are loaded into radial spokes, with
     the number of spokes given by min_particles_per_mode*(warpx.n_rz_azimuthal_modes-1).
 
+* ``<laser_name>.deposit_on_main_grid`` (`int`) optional (default `0`)
+    When using mesh refinement, whether the antenna that emits the laser
+    deposits charge/current only on the main grid (i.e. level 0), or also
+    on the higher mesh-refinement levels.
+
```

Comment:
From the code below, in Source/Particles/MultiParticleContainer.cpp,
```cpp
pp_lasers.queryarr("deposit_on_main_grid", tmp);
```
it seems like the input parameter should be `lasers.deposit_on_main_grid` (array of string, the names of the lasers to be deposited on the main grid), rather than `<laser_name>.deposit_on_main_grid` (int). Is that right or am I misinterpreting the code?

Reply:
the documented/2nd one (option per laser) is better/more common to what we do.

Reply:
As discussed today: I will change this in a follow-up PR for both `lasers.deposit_on_main_grid` and `particles.deposit_on_main_grid`.